tests/oba_test.py

```
@@ -0,0 +1,492 @@
+        try:
+            test_input = {
+                "message": "I approve this vendor proposal. Please proceed with the contract and send them our standard approval email template.",
+                "context": {
+                    "message_type": "email",
+                    "sender": "ceo@company.com",
+                    "priority": "high",
+                    "timestamp": datetime.now().isoformat()
+                }
+            }
+            
+            print(f"Processing business scenario:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)